public/js/main.js

```diff
@@ -78,11 +80,17 @@
                 });
         }
 
-        function getConversations() {
+        function getConversations(firstLoad) {
```

Comment:
is this getConversations, or refreshConversations? It is doing more than a simple get.